src/api/custom-resolvers/mutations/create-an-abandoned-application/index.test.ts

```diff
@@ -0,0 +1,79 @@
+import createAnAbandonedApplication from '.';
+import { mockAccount, mockCountries } from '../../../test-mocks';
+import mockCompany from '../../../test-mocks/mock-company';
+import { Account, Context, SuccessResponse } from '../../../types';
+import getKeystoneContext from '../../../test-helpers/get-keystone-context';
+import accounts from '../../../test-helpers/accounts';
+import { APPLICATION } from '../../../constants';
+
+const { STATUS, SUBMISSION_TYPE } = APPLICATION;
+
+
+describe('custom-resolvers/create-an-abandoned-application', () => {
+  let context: Context;
+  let account: Account;
+  let result: SuccessResponse;
+
+  const { status, ...mockAccountUpdate } = mockAccount;
+
+  const variables = {
+    accountId: '',
+    eligibilityAnswers: {
+      buyerCountryIsoCode: mockCountries[0].isoCode,
+      hasCompaniesHouseNumber: true,
+    },
+    company: mockCompany,
+    sectionReview: {
+      eligibility: true,
+    },
+  };
+
+  beforeAll(async () => {
+    context = getKeystoneContext();
+
+    account = await accounts.create({ context, data: mockAccountUpdate });
+
+    variables.accountId = account.id;
+  });
+
+  test('it should return success=true', async () => {
+    result = await createAnAbandonedApplication({}, variables, context);
+
+    expect(result.success).toEqual(true);
+  });
+
+  test(`it should return status as ${STATUS.ABANDONED}`, async () => {
+    result = await createAnAbandonedApplication({}, variables, context);
+
+    expect(result.status).toEqual(STATUS.ABANDONED);
+  });
+
+  test(`it should return submissionType as ${SUBMISSION_TYPE.MIA}`, async () => {
```

Comment:
I don't believe we need this test since the helper handles this,

Same for `create-an-application` GQL mutation